src/main/java/com/royalrangers/controller/LoginController.java

```diff
@@ -15,7 +17,8 @@
     private LoginService loginService;
 
     @PostMapping("/login")
-    public ResponseResult login(@RequestParam String email) {
+    public ResponseResult login(@RequestParam Map <String,Object> params) {
```

Comment:
Please revert your changes in this piece of code. For one request parameter, there are no need to create additional wrappers.